src/main/scala/uk/gov/nationalarchives/consignmentexport/S3Files.scala

```diff
@@ -7,17 +7,24 @@ import cats.effect.IO
 import cats.implicits._
 import io.chrisdavenport.log4cats.SelfAwareStructuredLogger
 import uk.gov.nationalarchives.aws.utils.S3Utils
+import uk.gov.nationalarchives.consignmentexport.Main.directoryType
 import uk.gov.nationalarchives.consignmentexport.Utils._
 import uk.gov.nationalarchives.consignmentexport.Validator.ValidatedFileMetadata
 
 import scala.language.postfixOps
 
 class S3Files(s3Utils: S3Utils)(implicit val logger: SelfAwareStructuredLogger[IO]) {
 
+  def createDirectories(files: List[ValidatedFileMetadata], consignmentReference: String, rootLocation: String) = {
```

Comment:
Maybe rename this: `createDownloadDirectories` as creating these to download the files from the s3 bucket to? Not sure?

Reply:
Yeah, that makes sense, I've changed it.